app/public/templates/scene.html

```diff
@@ -237,6 +248,10 @@
           };
           xhr.send(JSON.stringify(requestData));
       }
+      function getCookieScene(name) {
+          let r = document.cookie.match("\\b" + name + "=([^;]*)\\b");
+          return r ? r[1] : undefined;
+        }
```

Comment:
The nesting of function and the closing } aren't the same ( the } appears to be 2 spaces deeper ).